x/auction/keeper/rewards.go

```diff
@@ -18,11 +18,11 @@ import (
 
 func (k Keeper) FinalizeRewardsAuction() error {
 	now := k.ctx.BlockTime()
-	a, id := k.getRewardsAuction(0)
+	a, id := k.getRewardsAuction(k.currentRewardsAuctionID())
```

Comment:
every time we are query the initial acution

Reply:
please revert. 0 == latest auction.

Reply:
Ok